### PR TITLE
chore: Even cleaner dependencies

### PR DESCRIPTION
**Pull request by ukstv**
- Drop ancillary packages like js-sha256, @ethersproject/random, etc.
- Replace typestub-multihashes with multihashes.

Part 2 of #845. The final one.

### CODE REVIEW

**Review by stbrody — COMMENTED** [review]
looks good to me, but I'll leave it to Joel for final approval since he's more familiar with these libraries than I am.

File: packages/doctype-tile/package.json [hunk 1]
```diff
@@ -43,6 +40,7 @@
     "@typescript-eslint/eslint-plugin": "^4.6.0",
     "@typescript-eslint/parser": "^4.6.0",
     "babel-jest": "^26.6.3",
+    "dids": "^1.1.0",
```
Comment (oed) [inline packages/doctype-tile/package.json:43]: Can we please use 1.1.1?

File: packages/doctype-tile/src/tile-doctype.ts [hunk 2]
```diff
@@ -67,7 +67,7 @@ export class TileDoctype extends Doctype {
      */
     static async makeGenesis<T extends CeramicCommit>(params: DocParams, context: Context): Promise<T> {
         // If 'deterministic' is undefined, default to creating document uniquely
-        const unique = params.deterministic ? '0' : base64Encode(randomBytes(12))
+        const unique = params.deterministic ? '0' : uint8arrays.toString(randomBytes(12), 'base16')
```
Comment (oed) [inline packages/doctype-tile/src/tile-doctype.ts:70]: Why was this changed from base64 to base16?

**Review by oed — APPROVED** [review]
Lgtm mod comments.

File: packages/doctype-tile/src/tile-doctype.ts (hunk 2 above)
Comment (ukstv) [reply packages/doctype-tile/src/tile-doctype.ts:70]: so true. mea culpa

File: packages/doctype-tile/package.json (hunk 1 above)
Comment (ukstv) [reply packages/doctype-tile/package.json:43]: yup
